src/drivers/vmount/output_mavlink.cpp

```diff
@@ -54,7 +54,7 @@ OutputMavlink::OutputMavlink(const OutputConfig &output_config)
 
 int OutputMavlink::update(const ControlData *control_data)
 {
-	vehicle_command_s vehicle_command;
+	vehicle_command_s vehicle_command = {};
```

Comment:
As in further patch in this PR a new field is added to vehicle_command I choose to zeroing all vehicle_command to make it safe but I can just set this field in all vehicle_command.